vue-frontend/src/pages/jobs/[id].vue

```diff
@@ -382,7 +382,7 @@ export default {
   @apply tw-flex tw-items-baseline;
 }
 :deep(ul > li > a > svg) {
-  @apply tw-flex tw-items-center;
+  @apply tw-flex tw-items-center tw-w-[23px] tw-h-[25px] tw-ml-[1.5px];
```

Comment:
Why do we need this? Height and width are not used. Refer attachment below

![Screenshot from 2022-10-18 16-44-41](https://user-images.githubusercontent.com/69897605/196414952-b758c19d-1bfe-40d4-8ad1-6f53f320b298.png)